### PR TITLE
Added Verified Profile Service Verification page

### PR DESCRIPTION
**Pull request by shubhamsigdar1**
### Issue Closes https://github.com/Real-Dev-Squad/website-www/issues/733

Parent Issue Ticket: https://github.com/Real-Dev-Squad/website-www/issues/730

**Design Doc:** https://docs.google.com/document/d/1ra6Nagmfbywhc8-_fRtmv1K688uDwqLQTOH-Om2t7oE/edit?usp=sharing

### What is the change?

Provide a small description in points of what did you change

- Follow TDD
- Written test in which 
   - render h3 tag with text "Successful"
   - render p tag with text "Congratulations! Your Profile Service is Verified. Take the Next Step and Join Our Discord Server."
   - Render button with text "Next”
- In the app/components/identity-steps/step-seven.hbs, return UI get render when profile Status is "VERIFIED"

### Is Development Tested?

- [X] Yes
- [ ] No

### Is Under Feature Flag?

- [X] Yes
- [ ] No

### Test stats:

https://github.com/Real-Dev-Squad/website-www/assets/107163260/62b14b7e-8673-4212-b427-235d08e0a0f2

### Before / After Change Screenshots

**After Change Video**

https://github.com/Real-Dev-Squad/website-www/assets/107163260/7637bc1a-7884-4f11-bad5-3b277c795c82

### CODE REVIEW

File: app/components/identity-steps/step-seven.hbs [hunk 1]
```diff
@@ -1,36 +1,76 @@
...
+          </p>
+          <p data-test-verification-description-2>
+            Take the Next Step and Join Our Discord Server.
+          </p>
+        {{/if}}
+      </div>
+    {{/if}}
+  {{/each}}
+  <div class='verification-page__button'>
+    {{#if (eq this.login.userData.profileStatus 'PENDING')}}
```
[the remark this answers is not in the document]
Comment (shubhamsigdar1) [reply app/components/identity-steps/step-seven.hbs]: Done with changes

File: app/components/identity-steps/step-seven.hbs [hunk 2]
```diff
@@ -1,36 +1,76 @@
...
       <Reusables::Button
         @text='Refresh'
         @variant='dark'
         @onClick={{@handleRefresh}}
         @test='refresh'
         @type='button'
       />
-    </div>
-  {{/if}}
+    {{else if (eq this.login.userData.profileStatus 'BLOCKED')}}
```
[the remark this answers is not in the document]
Comment (shubhamsigdar1) [reply app/components/identity-steps/step-seven.hbs]: Done with changes

File: app/components/identity-steps/step-seven.hbs [hunk 3]
```diff
@@ -1,36 +1,76 @@
...
+      <Reusables::Button
+        @text='Verify Again'
+        @variant='dark'
+        @onClick={{@goToGenerateChaincodePage}}
+        @test='verify-again'
+        @type='button'
+      />
+    {{else if (eq this.login.userData.profileStatus 'VERIFIED')}}
```
[the remark this answers is not in the document]
Comment (shubhamsigdar1) [reply app/components/identity-steps/step-seven.hbs:66]: Done with changes

File: app/styles/identity-card.module.css [hunk 4]
```diff
@@ -1,3 +1,4 @@
+
```
[the remark this answers is not in the document]
Comment (shubhamsigdar1) [reply app/styles/identity-card.module.css]: Done with changes

File: app/components/identity-steps/step-seven.hbs (hunk 2 above)
[the remark this answers is not in the document]
Comment (shubhamsigdar1) [reply app/components/identity-steps/step-seven.hbs]: Done with changes

File: app/components/identity-steps/step-seven.hbs (hunk 3 above)
[the remark this answers is not in the document]
Comment (shubhamsigdar1) [reply app/components/identity-steps/step-seven.hbs]: Done with changes

File: app/styles/identity-card.module.css [hunk 5]
```diff
@@ -9,11 +10,36 @@
...
 }
 
+.ban-icon{
+    padding: 1.4rem 0 0 0.5rem;
+    color:  #ae03ba;
+}
+
+.square-check-icon{
+    padding: 1.4rem 0 0 0.5rem;
+    color:   #12f373;
```
[the remark this answers is not in the document]
Comment (shubhamsigdar1) [reply app/styles/identity-card.module.css]: Done with changes

File: app/styles/identity-card.module.css [hunk 6]
```diff
@@ -9,11 +10,36 @@
...
+        padding: 0.5rem 0 0 0.5rem;
+    }
+    
+    .ban-icon{
+        padding: 0.4rem 0 0 0.5rem;
+    }
+
+    .square-check-icon{
+        padding: 0.4rem 0 0 0.5rem;
+        color:   #12f373;
```
[the remark this answers is not in the document]
Comment (shubhamsigdar1) [reply app/styles/identity-card.module.css]: Done with changes

**Review by satyam73 — APPROVED** [review]
LGTM

**Review by heyrandhir — DISMISSED** [review]

**Review by sahsisunny — APPROVED** [review]

**Review by vinit717 — APPROVED** [review]

**Pull request merged**